metaflow/plugins/argo/argo_client.py

```diff
@@ -116,6 +116,7 @@ def trigger_workflow_template(self, name, parameters={}):
             "kind": "Workflow",
             "metadata": {"generateName": name + "-"},
             "spec": {
+                "serviceAccountName": "default-editor",
```

Comment:
Can you please create a JIRA Epic/Task to create a Metaflow Platform Plugin, such that we can create a Zillow plugin to specialize things (policies, etc) like this, and add a TODO for it here? (In this case it might just be a metaflow config).